crates/core/component/distributions/src/component.rs

```diff
@@ -143,13 +144,39 @@ trait DistributionManager: StateWriteExt {
         // Grab the ambient epoch index.
         let current_epoch = self.get_current_epoch().await?;
 
+        // New issuance for the current epoch.
         let new_issuance = self.compute_new_lqt_issuance(current_epoch).await?;
         tracing::debug!(
             ?new_issuance,
-            "computed new lqt reward issuance for epoch {}",
+            "computed new lqt reward issuance for current epoch {}",
             current_epoch.index
         );
-        Ok(self.set_lqt_reward_issuance_for_epoch(current_epoch.index, new_issuance))
+
+        // Retrieve the previous cumulative LQT issuance total from NV storage.
+        let previous_issuance = self
+            .get_cummulative_lqt_reward_issuance(current_epoch.index - 1)
+            .await
+            .ok_or_else(|| {
+                tonic::Status::not_found(format!(
```

Comment:
This is not an RPC, so we shouldn't return an RPC error. In fact, we shouldn't return an error at all. We do not want to interrupt component logic. Just default to zero.